README.md

```diff
@@ -1,67 +1,141 @@
-JavaScript Enclave Quickstart
-==============
+<center>
+    <img src="frontend/assets/img/near_logo_stack_wht.png" width="120" />
+</center>
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/near-examples/rust-status-message)
+# NEAR-SDK-JS Quick Start
 
-This is a template repo that you can use to quickstart your journey with writing smart contracts in JavaScript. The main branch contains a fully built frontend to support a simple coin flipping game alongside a skeleton smart contract found in the `src/index.js` file.
+> This example is designed to get you quickly writing and interacting with smart contracts on the NEAR blockchain using JavaScript! 🚀
 
-There are several other branches with different completed contracts that you can use as reference:
-- `coin-flip-skeleton` contains the skeleton code for the coin flipping game
-- `coin-flip-finished` contains the finished code for the coin flipping game
-- `coin-flip-hub` contains the finished code for the hub contract.
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/near-examples/near-sdk-js-quickstart)
 
-To test the fully working contract alongside the frontend, switch to the `coin-flip-finished` branch and follow the steps outlined in the [Quickstart](#quick-start) section.
+---
 
-### Prerequisites
+## Table of Contents
 
-In order to successfully complete this quickstart guide, you'll need to have a few things installed:
-- [Node.js](https://nodejs.org/en/about/) and [npm](https://www.npmjs.com/):
-```bash
-curl -sL https://deb.nodesource.com/setup_17.x | sudo -E bash -  
-sudo apt install build-essential nodejs
-PATH="$PATH"
-```
-Ensure that they are both installed by running a version check:
-```
-node -v
-npm -v
-```
+- [Requirements](#requirements)
+- [Quick Start](#quick-start)
+- [Exploring the Code](#exploring-the-code)
+  - [Smart Contract](#smart-contract)
+  - [Front end](#front-end)
+- [Example Game](#example-game)
+- [Help](#help)
+- [Feedback](#feedback)
 
-It's important to have the **newest** version of the [NEAR-CLI](https://docs.near.org/docs/tools/near-cli) installed such that you can make use of the JavaScript features. To install or update, run: 
+---
 
+## Requirements
+
+- [Node.js & npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- [near-cli](https://docs.near.org/docs/tools/near-cli) `v3.1.1` or greater
+
+```bash
+npm i -g near-cli
```

Comment:
I would mention that they need the most up-to-date version of the CLI otherwise the new commands won't be able to run.